en/docs/learn/exporting-user-data-from-wso2-is.md

```diff
@@ -0,0 +1,28 @@
+#Exporting User Data from WSO2 Identity Server
+
+1. Download **scim-bulk-user-export-tool** from [here]().
+2. Extract the downloaded .zip file.
+3. Run the `start.sh` script inside the extracted directory.
+
+    `sh start.sh`
+
+4. First you will be prompted to provide the host address of the WSO2 Identity Server instance.
+
+    !!! Note
+        
+        If you’re getting user data from the super-tenant you only need to provide the host address of the instance 
+        
+        (E.g. For a local instance `https://localhost:9443`). 
+        
+        If you’re getting data from a tenant you need append `t/<tenant-domain>` to host address 
```

Comment:
If you are getting data from a tenant, you need to append `t/<tenant-domain>` to the host address.

Reply:
Fixed 9c3530b